### DISCUSSION TITLE
cannot invite people in projects setting

### DISCUSSION THREAD

**Discussion by Mon-ius** [community/community#61575] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have a repo which is linked a project <https://github.com/orgs/xxxx/projects/1/settings/access>. 

The project was is set private. And I would like to add people to join. However, it does not pop up to send the invitation.

The repo is public for the org. So, the invitation should be fine to search the member who was outside the org.
 
<img width="1700" alt="image" src="https://github.com/community/community/assets/25671258/87ba0ca4-03e3-47a1-8436-4ad659431121">

<img width="1072" alt="image" src="https://github.com/community/community/assets/25671258/3280a7f2-751b-4644-839b-ae4d711ed8df">

**Comment by GhostOf0days** [comment]
Huh? You have the invite collaborators search bar. Type their username and click invite.

**Reply by Mon-ius** [reply]
Since the post result shows the empty return, dont think it actual send the invitation. I dont mind if u wanna join the test 🤗

**Comment by Mon-ius** [comment]
Double checked again that, if the An Orgnization Project cannot invite people outside from the Org. Only if the one is inside the org that can be invited and join the org project.

I tried first invite people inside the org and then the guy can be searched and shown inside the search bar.

Not sure if it's a feature 🤗
